src/config.rs

```diff
@@ -212,6 +212,14 @@ pub struct Config {
     #[serde(default)]
     #[serde(rename = "pypi-config")]
     pub pypi_config: PyPIConfig,
+
+    /// The location of the environments build by pixi
+    #[serde(default)]
+    #[serde(
+        rename = "target-environments-directory",
+        skip_serializing_if = "Option::is_none"
+    )]
```

Comment:
Now you have to add rename everywhere. Why not add `#[serde(rename_all="kebab-case")]` to the container (`Config`) and only add `alias` for "old" fields?